keymanager-lib/src/policy.rs

```diff
@@ -200,11 +218,11 @@ impl CachedPolicy {
     fn parse(raw: &[u8]) -> Result<Self> {
         // Parse out the signed policy.
         let untrusted_policy: SignedPolicySGX = cbor::from_slice(raw)?;
-        let policy = untrusted_policy.verify()?;
+        verify_and_trust_policy(&untrusted_policy)?;
```

Comment:
After verification the policy is no longer untrusted so the previous setup seemed nicer. You could do `let policy = untrusted_policy.policy;`.

Reply:
Had that but then I changed my mind, not sure why :smile: